### PR TITLE
映射文档 No.70

### PR DESCRIPTION
**Pull request by Hzwords**
完成分组No.70文档映射，新提交8个md文件。
torch.Tensor.nanmean paddle无对应api，未添加md文件
torch.frexp paddle无对应api，未添加md文件

### CODE REVIEW

**Comment by Hzwords** [comment]
@Tomoko-hjf 麻烦review一下，感谢

**Comment by Hzwords** [comment]
@Tomoko-hjf 麻烦有空review一下，谢谢

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+paddle.nn.functional.mish(x,
+                         name=None)
+```
+
+Pytorch 相比 Paddle 支持更多其他参数，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| input          | x         | 表示输入的 Tensor ，仅参数名不一致。                                     |
+| inplace     | -            | 表示在不更改变量的内存地址的情况下，直接修改变量的值，Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。 |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md]: 这里的inplace不能直接删去，可以通过 paddle.assign()赋值转写实现

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.eig.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+### 转写示例
+
+#### out：指定输出
+
+```python
+# Pytorch 写法
+torch.linalg.eig(t,out=y)
+
+# Paddle 写法
+paddle.assign(paddle.linalg.eig(t), y)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.eig.md]: tuple 应该是不能通过paddle.assign()赋值，可以改为
```python
torch.linalg.eig(t,out=(L, V))
L, V = paddle.linalg.eig(t)
```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.multi_dot.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+| tensors | x         | 表示输入的一个 tensor 列表 ，仅参数名不一致。                    |
+| out       | -       | 表示输出的 Tensor ， Paddle 无此参数，需要进行转写。 |
+
+### 转写示例
+
+#### out：指定输出
+
+```python
+# Pytorch 写法
+torch.linalg.multi_dot([[2,7], [1, 3]], out=y)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.multi_dot.md]: 这里的输入也用 x 代替吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/troch.nn.Mish.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+```python
+paddle.nn.Mish(name=None)
+```
+
+Pytorch 相比 Paddle 支持更多其他参数，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| inplace  | -        | 表示在不更改变量的内存地址的情况下，直接修改变量的值，Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。 |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/troch.nn.Mish.md]: 这里如果inplace=Fasle，可以写暂无转写方式，原地操作不能直接删除

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.nanmean.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+### 转写示例
+
+#### dytpe：指定数据类型
+
+```python
+# Pytorch 写法
+torch.nanmean(x, dim=-1, dtype=torch.float32)
+
+# Paddle 写法
+y = paddle.sparse.cast(x, value_dtype='float32')
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.nanmean.md]: 这里可以通过 y.astype()实现吗，可以的话这样就能写成一行 `paddle.nanmean(y.astype(),dim=-1)`

**Comment by Hzwords** [comment]
已修改，其中torch.nn.functional.mishapi中核查发现参数inplace默认值为False，是之前弄错了默认值，所以此次未修改参数映射部分内容，烦请再次review

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md (hunk 1 above)
Comment (Fancy-hjyp) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md]: 这里的inplace参数如果为True，则会在原地操作tensor，要想完全一致需要使用paddle.assign()赋值转写，所以需要添加转写示例
```python
# PyTorch 写法
y = torch.nn.functional.mish(x, True)

# Paddle 写法
y = paddle.nn.functional.mish(x)
paddle.assign(y, x)
```

Comment (Hzwords) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md]: 已修改

**Review by Fancy-hjyp — APPROVED** [review]
LGTM

**Comment by zhwesky2010** [comment]
torch.Tensor.nanmean
torch.frexp
这两个都有对应的API，如果在API文档里搜不到，可以直接import paddle跑下试试
API中文文档缺失可以提issue报bug

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+paddle.nn.functional.mish(x,
+                         name=None)
+```
+
+Pytorch 相比 Paddle 支持更多其他参数，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| input          | x         | 表示输入的 Tensor ，仅参数名不一致。                                     |
+| inplace     | -            | 表示在不更改变量的内存地址的情况下，直接修改变量的值，Paddle 无此参数，需要进行转写。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md]: 主要功能为节省显存，一般对网络训练影响不大，可直接删除。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/troch.nn.Mish.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+```python
+paddle.nn.Mish(name=None)
+```
+
+Pytorch 相比 Paddle 支持更多其他参数，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| inplace  | -        | 表示在不更改变量的内存地址的情况下，直接修改变量的值，Paddle 无此参数，暂无转写方式。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/troch.nn.Mish.md]: 主要功能为节省显存，一般对网络训练影响不大，可直接删除。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.nanmean.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+### 转写示例
+
+#### dytpe：指定数据类型
+
+```python
+# Pytorch 写法
+torch.nanmean(x, dim=-1, dtype=torch.float32)
+
+# Paddle 写法
+paddle.nanmean(x.astype('float32'),dim=-1)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.nanmean.md]: 指定输出数据类型
对返回值astype更严谨些

Comment (Hzwords) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.nanmean.md]: 但是这里torch.nanmean中描述是：If specified, the input tensor is casted to [dtype] before the operation is performed.  如果paddle那里在执行操作后才转换类型，会不会有影响操作结果的可能？

**Comment by Hzwords** [comment]
除nanmean外，均已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.nanmean.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+### [torch.Tensor.nanmean](https://pytorch.org/docs/1.13/generated/torch.Tensor.nanmean.html?highlight=nanmean#torch.Tensor.nanmean)
+
+```python
+torch.Tensor.nanmean(dim=None,
+                     keepdim=False,
+                     dtype=None,
+                     out=None)
+```
+
+### [paddle.Tensor.nanmean](暂无对应文档)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.nanmean.md:12]: 这个文档bug，可以提issue

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+Pytorch 相比 Paddle 支持更多其他参数，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| input          | x         | 表示输入的 Tensor ，仅参数名不一致。                                     |
+| inplace     | -            | 表示在不更改变量的内存地址的情况下，直接修改变量的值，主要功能为节省显存，一般对网络训练影响不大，Paddle 无此参数，可直接删除。 |
+
+### 转写示例
+
+#### inplcae
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md]: 删除了就不用转写了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.frexp.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
+## [ torch 参数更多 ]torch.frexp
+
+### [torch.frexp](https://pytorch.org/docs/1.13/generated/torch.frexp.html?highlight=frexp#torch.frexp)
+
+```python
+torch.frexp(input,
+            out=None)
+```
+
+### [paddle.frexp](暂无对应文档)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.frexp.md:10]: 可以提issue报文档bug

Comment (Hzwords) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.frexp.md:10]: 已提交issue

File: docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md (hunk 10 above)
Comment (Hzwords) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.mish.md]: 已修改

**Comment by Hzwords** [comment]
@zhwesky2010 请问什么时候可以合并？学院正在统计

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.nanmean.md (hunk 8 above)
Comment (zhwesky2010) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.nanmean.md]: OK 那就应该是astype输入值

**Review by zhwesky2010 — APPROVED** [review]
